spec/features/order_process/create_order_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Employee create order', type: :feature do
+  let!(:reward) { create(:reward, price: 1) }
+  let(:employee) { create(:employee, earned_points: 1) }
+
+  before do
+    sign_in employee
+    visit orders_path
+  end
+
+  it 'Create Order using erned points' do
+    expect(page).to have_content 'Number of earned points: 1'
+    click_link 'New Order'
+    expect(page).to have_current_path(new_order_path)
+    click_link 'Buy'
```

Comment:
it would be good to explicitly select a reward here (right now it is selected by default)